Dockerfile_build_ubuntu64

```diff
@@ -1,5 +1,5 @@
 ARG GO_VERSION
-FROM quay.io/influxdb/cross-builder:go${GO_VERSION}-latest
+FROM quay.io/influxdb/cross-builder:go${GO_VERSION}-326be487b2cd7a2df71767dab51395feec33911e
```

Comment:
NOTE: This is from a branch in the `edge` repo, pushed to confirm it's working as expected

Reply:
Will need updated after influxdata/edge#109 merges